packages/axios-extension/jest.config.js

```diff
@@ -4,6 +4,14 @@ module.exports = {
     },
     collectCoverage: true,
     collectCoverageFrom: ['src/**/*.ts'],
+    globals: {
+        'ts-jest': {
+            tsconfig: './test/tsconfig.json'
+        }
+    },
+    moduleNameMapper: {
+        '^@src/(.*)$': '<rootDir>/src/$1'
+    },
```

Comment:
Why do we need the changes in the `jest.config.js`?

Reply:
The change lets me use paths like this:
![image](https://user-images.githubusercontent.com/212316/168233516-51de8906-d6f8-495f-abb4-903373d794bc.png)

I prefer this over `import Foo from '../../src/foo'` -it's neater and lets me move the test around without breaking the import paths

Reply:
I agree, it is better than `../..` . Let me also introduce it in my open PR for the new middleware and then we can slowly use it everywhere. Or should we introduce a `jest.config.js` in the repository root and start extending it in each package?

Reply:
FWIW - IMO `../..` is clearer.

Reply:
Extending from a root `jest.config.js` sounds good, does it work well? I remember we tried something like this in tools-suite and gave up.

This works in concert with the path mapping in `tsconfig.json`.